src/TelnyxRTC/Verto.ts

```diff
@@ -130,6 +132,7 @@ export default class Verto {
     this.dialog = null;
     this.dialogs = {};
     this.params = params;
+    this.module = params.module;
```

Comment:
Can you throw an error early here if `module` is not `verto` or `telnyx_rtc`? That way we don't force the caller to wait until a method is called to throw a type error.

Reply:
I will do that